src/category_theory/morphism_property.lean

```diff
@@ -254,11 +254,27 @@ to isomorphisms in `D`. -/
 def is_inverted_by (P : morphism_property C) (F : C ⥤ D) : Prop :=
 ∀ ⦃X Y : C⦄ (f : X ⟶ Y) (hf : P f), is_iso (F.map f)
 
-lemma is_inverted_by.of_comp {C₁ C₂ C₃ : Type*} [category C₁] [category C₂] [category C₃]
+namespace is_inverted_by
+
+lemma of_comp {C₁ C₂ C₃ : Type*} [category C₁] [category C₂] [category C₃]
   (W : morphism_property C₁) (F : C₁ ⥤ C₂) (hF : W.is_inverted_by F) (G : C₂ ⥤ C₃) :
   W.is_inverted_by (F ⋙ G) :=
 λ X Y f hf, by { haveI := hF f hf, dsimp, apply_instance, }
 
+lemma op {W : morphism_property C} {L : C ⥤ D} (h : W.is_inverted_by L) :
+  W.op.is_inverted_by L.op :=
+λ X Y f hf, by { haveI := h f.unop hf, dsimp, apply_instance, }
+
```

Comment:
Can you add the `unop` version as well? Thanks!

Reply:
Thanks for the review. I have added `is_inverted_by.unop`. (However, the variants `is_localization.right/left_op` and `is_localization.unop` of `is_localization.op` shall be added only at a later stage, when more API on localization shall be available.)